src/App.tsx

```diff
@@ -3,17 +3,20 @@ import Chart from "./components/Chart";
 import StatedBalanceForm from "./components/StatedBalanceForm";
 import BalanceService from "./components/BalanceService";
 
+import { observer } from "mobx-react-lite"
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 import Navigation from "./components/pages/Navigation";
 import NameForm from "./components/pages/StatedNameForm";
 import GraphForm from "./components/pages/StatedGraphForm";
 
-const App = () => (
-    <div className="App">
+
+const App = observer(({ balances }) => (
```

Comment:
just pls rename it to balanceService.
App looks like nice factory pattern.

Reply:
ok done